TSC-Charter.md

```diff
@@ -176,10 +187,71 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public.
```

Comment:
I think that makes it harder on those that want to abstain.  If you vote then you are less identifiable.  I see that in a vote you can vote yes, no or abstain and we don't identify who voted yes or no but we are going to identify to abstained.